src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public abstract class Figure {
+
+    private Color color;
+    private double area;
+
+    public double getArea() {
+        return area;
+    }
+
+    public void setArea(double area) {
+        this.area = area;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    public String draw() {
+        return "";
```

Comment:
let's move this method to interface and make it abstract, we want our classes to be obligated to override it